lib/communityWallets.ts

```diff
@@ -1,4 +1,8 @@
 export default {
+    2640cd6d652ac94dc5f0963dcc00bcc7: {
```

Comment:
since this starts with a number, the whole string will need to be in quotes `'`

Reply:
Now fixed thanks